docs/contributing/code-style.md

```diff
@@ -365,20 +365,6 @@ Additional notes:
 
 ## Dangerous constructs in JavaScript and TypeScript
 
-### Do not access rows via `x.attr('zid')`; use `rows.id(x)`
-
-Our message row DOM elements have a custom attribute `zid` which
-contains the numerical message ID. **Don't access this directly as**
-`x.attr('zid')` ! The result will be a string and comparisons (e.g., with
-`<=`) will occasionally return the wrong result, just often enough
-to make a bug that's impossible to track down.
-
-You should instead use the `id` function from the `rows` module, as in
-`rows.id(x)`. This returns a number. Even in cases where you do want a
-string, use the `id` function, as it will simplify future code changes.
-In most contexts in JavaScript where a string is needed, you can pass a
-number without any explicit conversion.
```

Comment:
Just a small comment that generally I prefer to not use GitHub username mentions like `@timabbott` in commit messages because GitHub sends you annoying emails, sometimes multiple times for an individual commit as its gets added to forks, when doing so. They need a "silent mention" version of the feature :)

Reply:
Oh, that's completely annoying. Sorry about that!

Reply:
We have an item for this in the zulip-mobile style guide: https://github.com/zulip/zulip-mobile/blob/main/docs/style.md#mentioning-people

The point is generally applicable, so it'd probably be useful to put that someplace central.